contrib/mod_rewrite.c

```diff
@@ -1895,6 +1931,10 @@ static int rewrite_open_fifo(config_rec *c) {
     return -1;
   }
 
+  if (pr_fs_get_usable_fd2(&fd) < 0) {
+    return -1;
+  }
```

Comment:
Without this, the fd for a FIFO script is usually 2 (_i.e._ the same as stderr).  Not ideal.  And adding this bit o' code made the accompanying regression test pass.